.circleci/config.yml

```diff
@@ -164,59 +61,25 @@ common-steps:
         mkdir -p /tmp/workspace/${VERSION_CODENAME}
         mv ~/project/build/debbuild/packaging/*.deb /tmp/workspace/${VERSION_CODENAME}
 
-
   - &addsshkeys
     add_ssh_keys:
       fingerprints:
         - "e5:b5:6e:d0:4e:ce:52:40:33:30:5e:6f:c5:73:38:20"
 
-  - &setsdviewername
-    run:
-      name: Set package name to securedrop-workstation-viewer
-      command: |
-        mkdir ~/packaging
-        export PKG_NAME="securedrop-workstation-viewer"
-        # Enable access to this env var in subsequent run steps
-        echo $PKG_NAME > ~/packaging/sd_package_name
-        echo 'export PKG_NAME=$(cat ~/packaging/sd_package_name)' >> $BASH_ENV
-
-  - &setsdgrsecname
-    run:
-      name: Set package name to securedrop-workstation-grsec
-      command: |
-        mkdir ~/packaging
-        export PKG_NAME="securedrop-workstation-grsec"
-        # Enable access to this env car in subsequent run steps
-        echo $PKG_NAME > ~/packaging/sd_package_name
-        echo 'export PKG_NAME=$(cat ~/packaging/sd_package_name)' >> $BASH_ENV
-
-  - &setsdconfigname
-    run:
-      name: Set package name to securedrop-workstation-config
-      command: |
-        mkdir ~/packaging
-        export PKG_NAME="securedrop-workstation-config"
-        # Enable access to this env car in subsequent run steps
-        echo $PKG_NAME > ~/packaging/sd_package_name
-        echo 'export PKG_NAME=$(cat ~/packaging/sd_package_name)' >> $BASH_ENV
-
-  - &setsdkeyringname
-    run:
-      name: Set package name to securedrop-keyring
-      command: |
-        mkdir ~/packaging
-        export PKG_NAME="securedrop-keyring"
-        # Enable access to this env car in subsequent run steps
-        echo $PKG_NAME > ~/packaging/sd_package_name
-        echo 'export PKG_NAME=$(cat ~/packaging/sd_package_name)' >> $BASH_ENV
-
   - &setmetapackageversion
     run:
-      name: Get metapackage version via distribution changelog
+      name: Get and set metapackage version via distribution changelog
       command: |
         CURRENT_VERSION=$(grep -oP "\d+\.\d+\.\d+" ${PKG_NAME}/debian/changelog-buster | head -n1)
-        export VERSION_TO_BUILD="$CURRENT_VERSION"
+        if [[ "$SCHEDULE_NAME" != "" || "$CIRCLE_BRANCH" == "pleasebuildnightlies" ]]; then
```

Comment:
How about using an environment variable like `IS_NIGHTLY` that's `0` on `main` but `1` on `pleasebuildnightlies`, defined here in this configuration itself? That way we're not tied to any specific branch name (but do have to ensure we preserve that flag in `config.yml` on the branch).

Reply:
I came up with a better idea at 1am last night that should avoid all of this entirely, stay tuned...

Reply:
OK, there's now a `IS_NIGHTLY` environment variable that will be the string "nightly" during nightly builds, and empty string in regular builds. It's added as a parameter in the nightly workflow, so the job names have "nightly" in them (coming full circle!) and are otherwise identical except for that env variable.